packages/server-core/src/projects/project/project.test.ts

```diff
@@ -63,12 +63,10 @@ describe('project.test', () => {
     }
   })
 
-  before(async () => {
+  beforeEach(async () => {
     app = createFeathersKoaApp()
     await app.setup()
-  })
```

Comment:
do we need to setup and destroy engine after each test case?

Reply:
I think it's a good practice to ensure all state is wiped.